cmd/deploy.go

```diff
@@ -84,16 +81,11 @@ func init() {
 }
 
 // deployFn function runs deploy sub command.
-func deployFn(_ *cobra.Command, _ []string) error {
+func deployFn(cobraCmd *cobra.Command, _ []string) error {
```

Comment:
cobraCmd to not shadow package name. this is now where the root signal handled context lives and as such we gotta nicely pass it all along everywhere